azurerm/internal/services/netapp/resource_arm_netapp_snapshot.go

```diff
@@ -153,7 +158,45 @@ func resourceArmNetAppSnapshotRead(d *schema.ResourceData, meta interface{}) err
 		d.Set("location", azure.NormalizeLocation(*location))
 	}
 
-	return nil
+	return tags.FlattenAndSet(d, resp.Tags)
+}
+
+func resourceArmNetAppSnapshotUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).NetApp.SnapshotClient
+	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
```

Comment:
i realize the team is creating parsers for resource ids, would you like me to go ahead and create parsers for each of the netapp resoures or let that be addressed in another PR?

Reply:
It would be preferred to do it now but either or is fine 🙂

Reply:
sounds good, i'll get parsers added in